library_generation/utilities.sh

```diff
@@ -56,23 +56,18 @@ unzip_src_files() {
 
 # get gapic options from .yaml and .json files from proto_path.
 get_gapic_opts() {
-  local gapic_config
-  local grpc_service_config
-  local api_service_config
-  gapic_config=$(find "${proto_path}" -type f -name "*gapic.yaml")
-  if [ -z "${gapic_config}" ]; then
-    gapic_config=""
-  else
-    gapic_config="gapic-config=${gapic_config},"
-  fi
-  grpc_service_config=$(find "${proto_path}" -type f -name "*service_config.json")
-  api_service_config=$(find "${proto_path}" -maxdepth 1 -type f \( -name "*.yaml" ! -name "*gapic*.yaml" \))
+  local transport=$1
+  local rest_numeric_enums=$2
+  local gapic_yaml=$3
+  local grpc_service_config=$4
+  local service_yaml=$5
   if [ "${rest_numeric_enums}" == "true" ]; then
     rest_numeric_enums="rest-numeric-enums,"
   else
     rest_numeric_enums=""
   fi
-  echo "transport=${transport},${rest_numeric_enums}grpc-service-config=${grpc_service_config},${gapic_config}api-service-config=${api_service_config}"
+  gapic_yaml="gapic-config=${gapic_yaml},"
+  echo "transport=${transport},${rest_numeric_enums}grpc-service-config=${grpc_service_config},${gapic_yaml}api-service-config=${service_yaml}"
```

Comment:
```suggestion
  echo "transport=${transport},rest_numeric_enums=${rest_numeric_enums},service-config=${grpc_service_config},gapic_yaml=${gapic_yaml},service-yaml=${service_yaml}"
```

Reply:
Done.

`rest_numeric_enums` is not like other gapic options, where key and value separated by "=". When set to `false`, it can't present in the argument, otherwise the generator will treat it as `true`.

https://github.com/googleapis/sdk-platform-java/blob/4c9606323e88983151c71e6493868d739a395960/gapic-generator-java/src/main/java/com/google/api/generator/gapic/protoparser/PluginArgumentParser.java#L60-L62